libkbfs/bcache.go

```diff
@@ -308,9 +318,16 @@ func (b *BlockCacheStandard) DeletePermanent(id kbfsblock.ID) error {
 	block, ok := b.cleanPermanent[id]
 	if ok {
 		delete(b.cleanPermanent, id)
+		size := uint64(getCachedBlockSize(block))
 		b.bytesLock.Lock()
 		defer b.bytesLock.Unlock()
-		b.cleanTotalBytes -= uint64(getCachedBlockSize(block))
+		if b.cleanTotalBytes >= size {
+			b.cleanTotalBytes -= size
+		} else {
+			// In case the race mentioned in `PutWithPrefetch` causes us
+			// to undercut the byte count.
+			b.cleanTotalBytes = 0
+		}
```

Comment:
this should be a function since it's done twice. Something like `b.updateTotalBytes` or `b.updateTotalBytesLocked` depending on where you want the lock.